src/components/Select/Select.tsx

```diff
@@ -283,6 +282,11 @@ export const Select = React.forwardRef<HTMLButtonElement, SelectProps>(function
         return null;
     };
 
+    const uniqId = useUniqId();
+    const controlId = id ?? uniqId;
+    const selectId = `select-${controlId}`;
```

Comment:
Do we really need both ids `selectId` and `controlId`?

Reply:
Honestly, I do not know.

Seems like `SelectControlProps.selectId` was about only `aria-activedescendant` not `id` attribute. 
Also it is required property, while native `id` attribute is optional.